llvm/include/llvm/IR/ModuleSummaryIndex.h

```diff
@@ -538,10 +538,11 @@ class GlobalValueSummary {
   /// (either by the initializer of a global variable, or referenced
   /// from within a function). This does not include functions called, which
   /// are listed in the derived FunctionSummary object.
-  std::vector<ValueInfo> RefEdgeList;
+  SmallVector<ValueInfo, 0> RefEdgeList;
```

Comment:
nit: add a comment to explain the motivation to use `SmallVector<T, 0>` for future readers / maintainers.

Reply:
Fixed in the latest iteration.  Thanks!